src/views/AddAccountSelection.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div class="container pad-bottom">
+        <SmallPage>
+            <IdenticonSelector  v-if="!showLoader" :accounts="accounts" @identicon-selected="identiconSelected">
+                <PageHeader backArrow @back="back">Choose a new Account</PageHeader>
+            </IdenticonSelector>
+
+            <Loader v-else :title="title" :state="state" :lightBlue="true">
+                <template slot="success">
+                    <CheckmarkIcon/>
+                    <h1 class="title nq-h1">New address added<br>to your account.</h1>
+                </template>
+            </Loader>
+        </SmallPage>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { SmallPage, PageHeader, CheckmarkIcon } from '@nimiq/vue-components';
+import IdenticonSelector from '../components/IdenticonSelector.vue';
+import { AccountInfo } from '../lib/AccountInfo';
+import { State } from 'vuex-class';
+import { WalletStore } from '../lib/WalletStore';
+import { DerivedAddress } from '@nimiq/keyguard-client';
+import { ParsedSimpleRequest } from '../lib/RequestTypes';
+import { Address } from '../lib/PublicRequestTypes';
+import Loader from '../components/Loader.vue';
+import { Static } from '../lib/StaticStore';
+
+@Component({components: {Loader, SmallPage, PageHeader, IdenticonSelector, CheckmarkIcon}})
+export default class AddAccountSelection extends Vue {
+    @Static private request!: ParsedSimpleRequest;
+    @State private keyguardResult!: DerivedAddress[];
+
+    private showLoader: boolean = false;
+    private state: Loader.State = Loader.State.LOADING;
+    private title: string = 'Storing your address';
+
+    private get accounts(): AccountInfo[] {
+        return this.keyguardResult.map((address) => new AccountInfo(
+            address.keyPath,
+            '',
+            new Nimiq.Address(address.address),
+        ));
+    }
+
+    private back() {
```

Comment:
Private methods should start with an underscore....

Reply:
We had a decision on that.

Reply:
There are right now 10 `private _*` occurances vs 103 `private`s without `_` in `vue-components/src` and 13 `private _*` occurances vs 177 `private`s without `_` in `hub/src/*/*.vue` files.

You are welcome to also open an issue or PR to change all of them, but the stats are currently 10:1 in the non-underscore favor.

Reply:
Alright, I'll change it some day throughout the code base.